tests/plugins/selection_input.test.ts

```diff
@@ -644,4 +644,22 @@ describe("selection input plugin", () => {
     model.dispatch("CHANGE_RANGE", { id, rangeId: idOfRange(model, id, 0), value: "1:1" });
     expect(model.getters.getSelectionInput(id)[0].xc).toBe("1:1");
   });
+
+  test("consistent range colors upon refocusing multiple times", () => {
```

Comment:
this doesn't test what you fixed, actually. This issue was occuring when we would disable a selection input and enable a new one afterwards. I suggest that you have a look at the commands that are dispatched when you switch from one tab to the other in a chart sidepanel.

In a generic manner, try to first write the test to reproduced the issue before working on the fix ;-)

Reply:
Thanks for the tip, and yes you were correct I indeed missed one command which was mainly responsible for this behaviour, I have changed the test case, and now I think it tests the actual fix.